sale_order_transmit_method/models/sale_order.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    transmit_method_id = fields.Many2one(
```

Comment:
Could you change this field to computed stored readonly=False and change the onchange to compute method ? Thanks

Reply:
done